fn/func.go

```diff
@@ -15,3 +15,9 @@ func Reduce[T any, V any, S []V](s S, f Reducer[T, V]) T {
 
 	return accum
 }
+
+// Zero returns the zero value of the specified type.
+func Zero[T any]() T {
```

Comment:
I don't see this being used anywhere in the later commits? Can we remove it to not have a dependency on a new tag for the `fn` module?

Reply:
Yeah this can be removed. Leaving off the commit.